package.json

```diff
@@ -85,6 +85,7 @@
     "serve-favicon": "2.4.x",
     "underscore": "^1.8.3",
     "util": "^0.10.3",
-    "validator": "^7.0.0"
+    "validator": "^7.0.0",
+    "wmt-probation-rules": "ministryofjustice/wmt-probation-rules.git#tech/wmt-787/make-default-contracted-hours-more-generic"
```

Comment:
Yea, so once probation rules is merged in, I will change to the newest version before this PR gets merged.